Core/Source/MVVM/ViewModel.swift

```diff
@@ -54,10 +54,21 @@ public struct SecondaryActionInfo {
             case on
             case off
             case mixed
+
+            public init(isOn: Bool) {
```

Comment:
Since its basically the same effort and it looks a little cleaner lets just make State `ExpressibleByBooleanLiteral` so you can just pass in a true/false.